primitives/src/witness.rs

```diff
@@ -103,7 +103,7 @@ impl Witness {
     pub fn is_empty(&self) -> bool { self.witness_elements == 0 }
 
     /// Returns a struct implementing [`Iterator`].
-    #[must_use = "returned iterator should be used"]
+    #[must_use = "iterator adaptors are lazy and do nothing unless consumed"]
```

Comment:
This is not an adaptor though.

Reply:
Github just closed this?

Reply:
I just fixed it and force pushed?

Reply:
@tcharding I remember you had a similar issue last week or so, do you know why it did it?

Reply:
WTF, I can't reopen it. You probably have to create a new PR. :(

Reply:
Opened #3966

Reply:
I'm not sure why it closed, but the combination of closing and force-pushing is what causes it to be unopenable.